metadata-ingestion/setup.py

```diff
@@ -247,8 +247,8 @@ def get_long_description():
 }
 
 data_lake_profiling = {
-    "pydeequ>=1.0.1, <1.1",
-    "pyspark==3.0.3",
+    "pydeequ>=1.0.1",
+    "pyspark==3.3.2",
```

Comment:
It might still be good to keep this strictly pinned instead of >=

Reply:
[Spark 3.3.3](https://spark.apache.org/releases/spark-release-3-3-3.html) has already been released. Would it be better to fix the version to the 3.3x branch. i.e. compatible release
"pyspark~=3.3.0", so can upgrade Spark with any future micro releases in 3.3?

Reply:
make sense, thanks